app/register/controller.js

```diff
@@ -15,6 +18,12 @@ export default class RegisterController extends Controller {
     );
   }
 
+  @action
+  setChosenPhoto(chosenPhoto) {
+    this.photo = chosenPhoto;
+    this.model.photoURL = `/assets/images/imagesForUserRegistration/${this.photo}`;
```

Comment:
when you have data from src you can:
```suggestion
  setChosenPhoto(chosenPhoto) {
    this.model.photoURL = `${chosenPhoto}`;
```

Reply:
Nie mogę tak zrobić, ponieważ przekazuję nazwę owl.jpg do input photo - tak aby user widział, że wybrał jakieś tam zdjęcie :) dokonałam próby tej zmiany i wtedy się zorientowałam, że miałam plan dlatego nie wykorzystałam tej funkcjonalności :)